pkgs/by-name/ed/edido/package.nix

```diff
@@ -0,0 +1,19 @@
+{ lib
+, writeShellApplication
+, bc
+, diffutils
+, findutils
+, coreutils
+, firmwarePaths ? [
+    "/run/current-system/firmware"
+  ]
+}:
+writeShellApplication {
```

Comment:
This whole thing isn't necessary anymore, is it?

Reply:
I would like to keep it, to initially test config on a running system before rebooting, but I could simply move it to my own repo.

Reply:
I think it's fine if it's just for testing.